### DISCUSSION TITLE
Slit convolution with absorbance spectra?

### DISCUSSION THREAD

**Discussion by foff1nho** [radis/radis#704] [Ideas]
`apply_slit()` - can be applied to ***transmittance*** spectra, but not **_absorbance_**, see: the section on [spectral arrays](https://radis.readthedocs.io/en/latest/spectrum/spectrum.html#spectral-arrays) which describes for which spectra types a convolution with a slit function is applied.

Is it possible to convolve a slit function with an absorbance spectrum, or is this not physically possible?

**Comment by erwanp** [comment]
sorry for late answer : it was not implemented because it does not fit an experimental use case, but it could still be done in your script a posteriori
